gateway-service/src/user/user.controller.ts

```
@@ -115,4 +115,52 @@ export class UserController {
   }
+  @ApiTags('Course Video')
+  @Get('coursevideos')
+  async getVideos(@Headers('authorization') token: string) {
+    try {
+      const jwtPayload = this.jwtService.verify(token, {
+        secret: process.env.JWT_SECRET,
+      });
+
+      return this.userService.getCourseVideos(jwtPayload.userId);
+    } catch (error) {
+      throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)